### ISSUE TITLE
You might be aware, but I just wanted to let you know that uSync Publisher's integration in Umbraco 15.1 RC1 is broken. In dev console an error is logged about a missing export.

### ISSUE THREAD

**Issue by KevinJump** [Jumoo/uSync.Complete.Issues#251]
You might be aware, but I just wanted to let you know that uSync Publisher's integration in Umbraco 15.1 RC1 is broken. In dev console an error is logged about a missing export.

_Originally posted by @PhyxionNL in https://github.com/Jumoo/uSync.Complete.Issues/issues/249#issuecomment-2507416792_

**Comment by KevinJump** [comment]
This is a breaking change in a point release. 😞 - so I am going to follow this up with the Umbraco team - although hopes are low.

**Comment by PhyxionNL** [comment]
Yeah, thought so ☹️ There's no way around it?

**Comment by KevinJump** [comment]
At the moment. no because there is no public npm package for v15.1 so i can't even update the client code and build a v15.1 version of the package.

**Comment by KevinJump** [comment]
https://github.com/umbraco/Umbraco-CMS/issues/17678

**Comment by PhyxionNL** [comment]
> At the moment. no because there is no public npm package for v15.1 so i can't even update the client code and build a v15.1 version of the package.

Probably because they moved it back into main repo. You might be able to use [GitPkg](https://gitpkg.vercel.app/about):
npm install 'https://gitpkg.vercel.app/umbraco/Umbraco-CMS/src/Umbraco.Web.UI.Client?release/15.1'

**Comment by KevinJump** [comment]
yes, I am banging on some doors because it should be public,  .. just making the usual nuisance of myself.

**Comment by KevinJump** [comment]
fixed before v15.1 release
